pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -354,6 +411,14 @@ in buildPythonPackage {
 
   src = bazel-build.python;
 
+  postPatch = ''
```

Comment:
It's probably clear from the regexes themselves, but consider still adding a comment on why is this patching needed? (= versions relaxation due to mismatch between what Nixpkgs has and TF requires?)

Reply:
Done